python-pytest.el

```diff
@@ -479,7 +479,7 @@ When present ON-REPLACEMENT is substituted, else OFF-REPLACEMENT is appended."
   :class 'transient-switches
   :key "-v"
   :argument-format "%s"
-  :argument-regexp "\\(--verbose\\|--verbose --verbose\\)"
+  :argument-regexp "^\\(--verbose --verbose\\|--verbose\\)$"
```

Comment:
ah, good catch. is the reordering necessary? i guess not? groups match from left to right, but with the anchoring in place it can stay in the same order as `:choices` below?

Reply:
Indeed the reordering is not necessary. I reverted it in the amended commit.
(The reordering was my first try  before I found the actual cause of this issue).